s3/sc-product-s3-synapse.yaml

```diff
@@ -0,0 +1,48 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: S3 Synapse ServiceCatalog product.
+Parameters:
+  PortfolioProvider:
+    Type: String
+    Description: Owner and Distributor Name
+  LaunchConstraintARN:
+    Type: String
+    Description: ARN of the launch constraint role for S3 products.
+  PortfolioId:
+    Type: String
+    Description: The ServiceCatalog portfolio this product will be attached to.
+  RepoRootURL:
+    Type: String
+    Description: Root url for the repo containing the product templates.
+  ProductVersionName:
+    Type: String
+    Description: The product version name in the service catalog
+Resources:
+  scs3product:
+    Type: AWS::ServiceCatalog::CloudFormationProduct
+    Properties:
+      Name: Amazon S3 Synapse Bucket
+      Description: This product builds an Amazon AWS S3 bucket with private access for Synapse
+      Owner: !Ref 'PortfolioProvider'
+      Distributor: !Ref 'PortfolioProvider'
+      SupportDescription: Operations Team
+      SupportEmail: support@yourcompany.com
+      AcceptLanguage: en
+      SupportUrl: http://helpdesk.yourcompany.com
+      ProvisioningArtifactParameters:
+        - Description: baseline version
+          Info:
+            LoadTemplateFromURL: !Sub '${RepoRootURL}s3/sc-s3-synapse-ra.yaml'
+          Name: !Ref 'ProductVersionName'
+  Associates3:
+    Type: AWS::ServiceCatalog::PortfolioProductAssociation
+    Properties:
+      PortfolioId: !Ref 'PortfolioId'
+      ProductId: !Ref 'scs3product'
+  constraintec2linux:
```

Comment:
should have more appropriate name

Reply:
this was just copied from https://github.com/Sage-Bionetworks/scipoolprod-sc-lib-infra/blob/master/s3/sc-product-s3-private.yaml#L39

Reply:
understood but looks like a mistake on their part